armi/reactor/tests/test_composites.py

```diff
@@ -387,6 +387,16 @@ def test_getFuelMass(self):
 
         self.assertEqual(fuelMass, a.getFuelMass())
 
+    def test_getNeutronEnergyDepositionConstants(self):
```

Comment:
Perhaps you could add a short comment to this one and the next about _why_ an error is raised from this call? Because at first glance, it doesn't look like something I would expect to fail.

Reply:
Done.

The problem is the little test reactor doesn't have a cross-section library loaded. But loading one into it would break a bunch of other tests.